dynflow.gemspec

```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |s|
   s.add_dependency "multi_json"
   s.add_dependency "apipie-params"
   s.add_dependency "algebrick", '~> 0.7.0'
-  s.add_dependency "concurrent-ruby", '~> 1.0'
-  s.add_dependency "concurrent-ruby-edge", '~> 0.2.0'
+  s.add_dependency "concurrent-ruby", '= 1.1.0.pre2'
+  s.add_dependency "concurrent-ruby-edge", '= 0.4.0.pre2'
```

Comment:
is this still needed?

Reply:
I guess it's needed until the c-r 1.1 is in pre-release

Reply:
We still use Actors which live in -edge

Reply:
I see :+1: